src/domain/app/footer/FooterCategories.tsx

```diff
@@ -6,35 +7,58 @@ import CategoryFilter from '../../../common/components/category/CategoryFilter';
 import { Category } from '../../../common/types';
 import useLocale from '../../../hooks/useLocale';
 import scrollToTop from '../../../util/scrollToTop';
-import { EVENT_DEFAULT_SEARCH_FILTERS } from '../../eventSearch/constants';
+import { EventRouteProp } from '../../event/types';
 import {
+  COURSE_DEFAULT_SEARCH_FILTERS,
+  EVENT_DEFAULT_SEARCH_FILTERS,
+} from '../../eventSearch/constants';
+import { CategoryOption, Filters } from '../../eventSearch/types';
+import {
+  getCourseCategoryOptions,
   getEventCategoryOptions,
   getSearchQuery,
 } from '../../eventSearch/utils';
 import { ROUTES } from '../routes/constants';
 import styles from './footerCategories.module.scss';
 
-const FooterCategories: FunctionComponent = () => {
+interface FooterProps {
+  route: EventRouteProp;
```

Comment:
`export type EventRouteProp = typeof ROUTES.EVENTS | typeof ROUTES.COURSES;`

would do the trick in EventRouteProp

Reply:
fixed